util/util.py

```diff
@@ -179,3 +180,20 @@ def remove_ctf_by_channel_id(database, ctf_channel_id):
     ctf = ctfs[ctf_channel_id]
     ctfs.pop(ctf_channel_id)
     pickle.dump(ctfs, open(database, "wb"))
+
+
+def parse_user_id(user_id):
+    """
+    Parses a user_id, removing possible @-notation and make sure it's uppercase
```

Comment:
You're gonna kill me for all these nitpicks, but: 'Parse' vs. 'Parses'

Reply:
Nah, it's right, just not used to python stlyle / docstrings. But I would point those things out, too in a code review, if they violate a predefined style guide.